parent/pom.xml

```diff
@@ -585,7 +586,19 @@
                         <dependency>
                             <groupId>org.apache.groovy</groupId>
                             <artifactId>groovy</artifactId>
-                            <version>4.0.27</version>
+                            <version>${cxf.groovy.version}</version>
+                        </dependency>
+                    </dependencies>
+                </plugin>
+                <plugin>
+                    <groupId>org.codehaus.gmaven</groupId>
```

Comment:
@ppalaga why do we need `groovy-maven-plugin` plugin? AFAIK it is not supported and will be problematic (potentially) for Maven 4 support.

Reply:
> why do we need groovy-maven-plugin plugin?

The first solution I could figure out that does not change the timestamp on the local file when there is no content change. Those timestamps might be important for incremental builds. 
Ant plugin could perhaps be used to achieve the same with a bit more code - should I try?

> AFAIK it is not supported

In which sense not supported?

Reply:
I see. gmavenplus has some recent releases: https://mvnrepository.com/artifact/org.codehaus.gmavenplus/gmavenplus-plugin
Would that one be OK?

Reply:
Ouch, I just found out, how many changes the suffix file is receiving https://github.com/publicsuffix/list/commits/main/public_suffix_list.dat
It is nearly every day.
I am afraid, the approach proposed in this PR would not be sustainable.

It would actually be enough to update the list shortly before releasing CXF. @ffang I wonder whether there is any release automation in place for CXF? Is there a CI job or a script? If so, the suffix list update could be added there.

Reply:
Hi @ppalaga ,

No such automation before release where we can put the [public_suffix_list.dat](https://github.com/publicsuffix/list/commits/main/public_suffix_list.dat) update in.

But as we need to do manual steps before each release anyway, like update release notes in distribution/src/main/release/release_notes.txt, so we can also update the offline public_suffix_list.dat before each release.

Cheers
Freeman